fastapi_jwks/injector/payload_injector.py

```diff
@@ -3,9 +3,23 @@
 from pydantic import BaseModel
 from starlette.requests import Request
 
+from fastapi_jwks.models.types import JWTTokenInjectorConfig
+
 TypeT = TypeVar("TypeT", bound=BaseModel)
+RawTokenT = TypeVar("RawTokenT")
 
 
 class JWTTokenInjector(Generic[TypeT]):
+    def __init__(self, config: JWTTokenInjectorConfig | None = None):
+        self.config = config or JWTTokenInjectorConfig()
+
     async def __call__(self, request: Request) -> TypeT:
-        return request.state.payload
+        return getattr(request.state, self.config.payload_field)
+
+
+class JWTRawTokenInjector(Generic[RawTokenT]):
+    def __init__(self, config: JWTTokenInjectorConfig | None = None):
+        self.config = config or JWTTokenInjectorConfig()
+
+    async def __call__(self, request: Request) -> RawTokenT:
+        return getattr(request.state, self.config.token_field)
```

Comment:
Could you also add sections for `Contributing` and `Licence` like we have for Kpops: https://github.com/bakdata/kpops?tab=readme-ov-file#contributing